content/1_docs/1_guide/20_kirby-and-privacy/guide.txt

```diff
@@ -0,0 +1,63 @@
+
+Kirby sites usually do not store or process any personal data of your site visitors. However, here is a list of Kirby features that process data if you use them and some general information about where personal data might get stored.
+
+## User handling
+
+User data gets stored in account files within the `site/accounts` directory. Kirby will create a session cookie to keep track of the currently logged in user if you use a login form in your site’s frontend or if you use the Panel. 
+
+The user credentials are transmitted via HTTP. Therefore, we strongly recommend to use TLS encryption for your sites to protect the passwords and other personal data of your visitors and users.
+
+## Panel
+
+The Panel sets a session cookie for every visitor who uses the Panel to keep track of the currently logged in user. Additionally, the Panel stores unsaved changes in `localStorage`.
+
+## Licensing
+
+To register your license, the Panel connects to our licensing server to verify your license. When you do so, the license server stores the domain for which the license is registered on our license server.
+
+## Brute-force protection
+
+To protect the Panel login against brute-force attacks, Kirby temporarily stores a shortened SHA256 hash of the IP address on login failures. This hash cannot be converted back to the raw IP address. You can control the number of possible trials before brute-force protections kicks in and the time span for which this data is stored in your (link: docs/reference/system/options/auth text: config settings).
+
+## Language detection
+If you use the (link: docs/reference/system/options/languages text: `languages.detect` option) on a multilang site, Kirby also creates a session cookie to keep track of the visitor’s language. This option is disabled by default.
+
+## `csrf()` helper
+If you use the (link: docs/reference/templates/helpers/csrf text: `csrf()` helper), Kirby will create a session cookie so that the helper can validate the CSRF token in a later request.
+
+## Plugins and custom code
+Your sites may store or process additional personal data depending on the Kirby plugins and custom code you are using. For example, some plugins like contact form plugins also use sessions for technical reasons.
+
+## Contact forms
+Your sites also process personal data once a contact form is submitted, a blog comment gets stored or files get uploaded by visitors. The same also applies to similar custom site features. The data you store in your content files may also contain personal data.
+
+## Hosting providers
+Data might also get stored and processed by your hosting provider. What sort of data they store and process depends on your contract with the hosting provider.
+
+
```

Comment:
Maybe a horizontal line here to separate the following paragraph from the text (or an info box for that paragraph).